### PR TITLE
Add compatible versions of ipython, ipython-genutils and ipdb for ipd…

### PR DESCRIPTION
**Pull request by mkangia**
…b to work correctly

@benrudolph @proteusvacuum 

cc: review buddy @biyeun

### CODE REVIEW

File: requirements/dev-requirements.txt [hunk 1]
```diff
@@ -15,6 +15,8 @@ fixture
 sniffer
 psutil  # for memory profiling
 django-extensions
-ipdb
+ipython==4.2.1
+ipython-genutils
```
Comment (mkangia) [inline requirements/dev-requirements.txt]: Thanks @benrudolph. Missed adding version for ipython-genutils.. Add now..

**Comment by proteusvacuum** [comment]
@mkangia what was the issue you were having with ipython v5? It is working for me, and has lots of nice features like syntax highlighting.

**Comment by proteusvacuum** [comment]
(sorry, didn't realize the current version was v5)

**Comment by benrudolph** [comment]
yeah, so it _almost_ works. for me and manish it crashes with ipdb. what i've experienced it that tab completion stops working in ipdb when you're on v5 of ipython. when you upgrade ipdb, the debugger crashes 😞

**Comment by benrudolph** [comment]
guessing it'll probably be fixed soon...

**Comment by proteusvacuum** [comment]
huh, I wonder if that is a macOS thing? I think it was working fine for me.

**Pull request merged**